src/ssu.cpp

```diff
@@ -4,14 +4,20 @@
 #include <QDBusPendingReply>
 #include <QDebug>
 
+#if defined(SAILFISHOS_TARGET_VERSION) && SAILFISHOS_TARGET_VERSION >= 40600
```

Comment:
IIRC it shall be `46000` to correspond to "4.6.0.00", but maybe I remember that incorrectly.

Reply:
See the end of https://build.sailfishos.org/project/prjconf/sailfishos:4.6

Reply:
I see, also at the start of older `prjconf`s, e.g. https://build.sailfishos.org/project/prjconf/sailfishos:3.1.0.12